drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4207,9 +4345,13 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 
 		render_list.clear();
 		_fill_render_list(p_cull_result, p_cull_count, true, false);
-		render_list.sort_by_key(false);
+		if (use_oq) {
+			render_list.sort_by_depth(false);
+		} else {
+			render_list.sort_by_key(false);
+		}
```

Comment:
Well, I need to do this since AFAIK I don't have a depth prepass on mobile.

On desktop I managed to get a nice performance boost during the depth pass since the bounding boxes only need 8 vertices. In badly optimized scenes that can make a suprisingly big difference.